.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+## Description
+Please provide a short summary of the changes and the motivation behind them.
```

Comment:
nit: Surround in XML comment, so that this isn't included in the PR by default.

```Markdown
<!-- Please provide a short summary of the changes and the motivation behind them. -->
```